### DISCUSSION TITLE
What about a Mindmap Integration in Hyas / Doks ?

### DISCUSSION THREAD

**Discussion by cnichte** [gethyas/hyas#464] [Ideas]
Hi, Mindmap integrations have often been discussed - also in the Hugo forum. In most cases, the approaches failed because they involved proprietary data formats. With [Markmap](https://markmap.js.org) there is a nice conversion from Markdown into a mindmap, which I also use in VS-Code and Obsidian. 

* https://github.com/markmap/markmap

So far I copy screenshots over, but i could imagine that an integration in Hyas / Doks actually makes sense here...?

**Comment by h-enk** [comment]
What about [PlantUML's mindmap](https://getdoks.org/docs/built-ins/diagrams/#plantuml)?
